src/relay-proxy/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+name: 'Overview'
+route: /docs/relay-proxy
+menu: Relay proxy
+---
+
+import { Link, Title, Code, TextBlock } from 'components'
+
+<Title>Smartlook Relay Proxy</Title>
+
+This service relays HTTP requests from your subdomain to
+Smartlook's recording domains, allowing you to bypass potential browser settings or extensions used by your visitors configured to limit data capture.
+
+<TextBlock kind="important">
+	Only web recording is currently supported. Support for Mobile SDKs will be
+	added soon.
```

Comment:
```suggestion
	Only web recording is currently supported.
```
https://github.com/smartlook/smartlook-relay-proxy/commit/6ffcb941c2912cb0127d8c1b3f88f3426d97f7a3
Dnes rano jsme zrovna resili, ze tu podporu pro Mobile SDKs odlozime na neurcito. Co tak vime, tak zatim zadny zakaznik nepouziva ani tu webovou😕 Takze dokud na to neprijde request od zakazniku/produktu, tak se tim nebudeme zaobirat